app/services/sync_service/blogs.rb

```diff
@@ -3,8 +3,17 @@
 require "open-uri"
 
 class SyncService::Blogs
-  def self.call(params = {})
-    new(params).sync_blog_posts
+  def self.call(blog_id: nil)
+    unless blog_id
+      blog_ids = Blog.all.map { |b| b.id }
+    else
+      blog_ids = [ blog_id.to_i ]
+    end
+    blog_ids.each do |blog_id|
+      blog = Blog.find(blog_id)
+      blog_feed = OpenStruct.new(feed_path: blog.base_url + blog.feed_path, id: blog_id)
```

Comment:
I'm confused by using OpenStruct here? Shouldn't the Blog model be sufficient?

Reply:
@nomadicoder ^

Reply:
@bibliotechy Mea Culpa. OpenStruct instantiation removed.